daemon/container.go

```diff
@@ -227,6 +227,21 @@ func (container *Container) GetRootResourcePath(path string) (string, error) {
 	return symlink.FollowSymlinkInScope(filepath.Join(container.root, cleanPath), container.root)
 }
 
+func (container *Container) ExportRw() (archive.Archive, error) {
```

Comment:
Yes, this is a required function.  It was taken as-is from the original implementation in container.go and split into container_unix.go back in April as part of https://github.com/docker/docker/commit/b9e4b95788e0d8eae5363b82f301dafb9ea687f7.  This change simply moves it back into container.go now that Windows graphdriver can support the same export semantics.